WordPress/Classes/Services/PostService+Likes.swift

```diff
@@ -52,19 +52,10 @@ extension PostService {
      
      @param postID  The ID of the post to fetch likes for.
      @param siteID  The ID of the site that contains the post.
-     @param before  Filter results to likes before this date/time. Optional.
      */
-    func likeUsersFor(postID: NSNumber, siteID: NSNumber, before: String? = nil) -> [LikeUser] {
+    func likeUsersFor(postID: NSNumber, siteID: NSNumber) -> [LikeUser] {
```

Comment:
Optional: since we are refactoring this part, maybe we can consolidate the two `likeUsersFor` methods into one and construct the predicate based on if there is an `after: Date` or not?

Reply:
Hehehe. You mean the bit I removed? Yea... that makes sense. 😄 Will do! Thanks!